great_expectations/cli/cli.py

```diff
@@ -321,6 +321,74 @@ def render(render_object):
     print(DefaultJinjaPageView.render(model))
 
 
+@cli.command()
+@click.option(
+    '--target_directory',
+    '-d',
+    default="./",
+    help='The root of the project directory where you want to initialize Great Expectations.'
+)
+def check_config(target_directory):
+    """Check a config and raise helpful messages about antiquated formats."""
+    cli_message("Hi! Let's check your config files for validity.")
+
+    do_config_check(target_directory)
+
+
+def do_config_check(target_directory):
+    try:
+        context = DataContext(
+            context_root_dir="{}/great_expectations/".format(target_directory))
+        config = context.get_project_config()
+        check_for_obsolete_config_file(config)
+        cli_message("Your config file appears valid!")
+    except (
+            ge_exceptions.InvalidConfigurationYamlError,
+            ge_exceptions.InvalidTopLevelConfigKeyError,
+            ge_exceptions.MissingTopLevelConfigKeyError,
```

Comment:
In the typical case of check-config being executed in a pre-0.8.0 project, this is the message that is displayed:
```
You are missing a required key: `ge_config_version`
    Please ensure that it is in your config file.
```

While it is technically correct, I think we should treat ge_config_version as a special key  and interpret its absence as ge_config_version=0.

Reply:
1. I think you are right to press me on this. I'll do it.
2. You are spot on with this one! I'll implement

Reply:
@Aylr Thank you!

Reply:
Both are implemented and tested.